src/util-exception-policy-types.h

```diff
@@ -40,6 +40,19 @@ enum ExceptionPolicy {
  * "tcp.reassembly_exception_policy.drop_packet" + 1 */
 #define EXCEPTION_POLICY_COUNTER_MAX_LEN 45
 
+/** Possible scenario/ config settings for exception policies */
+enum ExceptionPolicyTargetValues {
+    EXCEPTION_TARGET_FLAG_NONE = 0,
+    EXCEPTION_TARGET_FLAG_DEFRAG_MEMCAP = BIT_U16(0),
+    EXCEPTION_TARGET_FLAG_SESSION_MEMCAP = BIT_U16(1),
+    EXCEPTION_TARGET_FLAG_REASSEMBLY_MEMCAP = BIT_U16(2),
+    EXCEPTION_TARGET_FLAG_FLOW_MEMCAP = BIT_U16(3),
+    EXCEPTION_TARGET_FLAG_MIDSTREAM = BIT_U16(4),
+    EXCEPTION_TARGET_FLAG_APPLAYER_ERROR = BIT_U16(5),
```

Comment:
Haven't checked the PR. Not sure if these bitfields are to be used as enum values, I've only seen them as macros. Keeping that aside, do we expect more fields here? Seems like these many could fit in U8.